Source/DafnyPipeline.Test/RelativePaths.cs

```diff
@@ -10,5 +10,14 @@ public class RelativePaths {
     public void Test() {
       Assert.Equal(0, DafnyDriver.ThreadMain(new string[] { "/spillTargetCode:3", "warnings-as-errors.dfy" }));
     }
+
+    [Fact]
+    public void TestFileConversion() {
+      var filePath = @"file:\c:\Users\xxx\Documents\with%20space\test.dfy";
+      var filePath1 = @"file:c:\Users\xxx\Documents\with%20space\test.dfy";
+      var expected = @"c:\Users\xxx\Documents\with space\test.dfy";
```

Comment:
Please add corresponding tests for [POSIX paths](https://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap03.html#tag_03_266) too.